cassava-megaparsec.cabal

```diff
@@ -28,7 +28,7 @@ library
   build-depends:      base             >= 4.8   && < 5.0
                     , bytestring       >= 0.9   && < 0.11
                     , cassava          >= 0.4.2 && < 0.6
-                    , megaparsec       >= 7.0   && < 8.0
+                    , megaparsec       >= 7.0   && < 9.0
```

Comment:
@Jagl257 Can we update the changelog and the cabal version so that we can upload the package to hackage?

Reply:
Ok, @CristhianMotoche